### PR TITLE
Fix #996 - Stop alias name from flickering

### LINKED ISSUE #996: An alias name that is longer than the initial empty name field flickers when hovering over it
#### Build:
- Stage;

#### Affected Platforms:
- Windows 10;

#### Browsers tested on:
- Firefox, latest version;

#### Prerequisites:
- Be signed in into a Firefox Relay account;
- Have a few aliases created;
- Have the stage version of the Private Relay add-on installed

#### Steps to reproduce:
1. Go to https://stage.fxprivaterelay.nonprod.cloudops.mozgcp.net/accounts/profile/ ;
2. Name one of the created aliases with a maximum length name;
3. Hover over the set name field, from left to right, until exiting the name’s space;

#### Expected result:
- The alias name is shown without issue and the field highlights;

#### Actual result:
- The alias name field flickers and cuts the name;

#### Notes:
- The name field cuts back to its initial size then back to the full size of the set name when flickering;

alias name bar

### PR DESCRIPTION
**Pull request by codemist**
**Flickering on hover has stopped, check demo:**

https://user-images.githubusercontent.com/13066134/128577819-4b11e632-c05a-488b-9ab5-9c2a344097fd.mov

### CODE REVIEW

File: static/scss/partials/dashboard.scss [hunk 1]
```diff
@@ -198,6 +198,12 @@
         display: none;
     }
 
+    .relay-email-address-label:placeholder-shown {
+        color: $color-grey-50;
+        opacity: 1;
+        width: 192px;
```
Comment (codemist) [inline static/scss/partials/dashboard.scss:204]: Magic Number specifically for the placeholder.

File: static/scss/partials/main.scss [hunk 2]
```diff
@@ -809,13 +805,6 @@ input.relay-email-address-label {
   width: 200px;
 }
 
-input::placeholder,
-.relay-email-address-label:placeholder-shown {
-  color: $color-grey-50;
-  opacity: 1;
-  width: 192px;
-}
-
```
Comment (codemist) [inline static/scss/partials/main.scss:818]: Moved this into `dashboard.scss` for clarity

File: static/scss/partials/main.scss [hunk 3]
```diff
@@ -849,8 +838,7 @@ input:focus::placeholder {
 }
 
 .relay-email-address-label:hover {
-  background: $color-grey-20;
-  width: 192px;
+  background: $color-grey-10;
```
Comment (codemist) [inline static/scss/partials/main.scss:841]: Opted for a lighter grey, it looks closer to the input field on Figma (lemme know if you agree!)

**Review by codemist — COMMENTED** [review]
Started a self review for clarification

**Review by maxxcrawford — CHANGES_REQUESTED** [review]
I think this solution may be over-engineered. When testing, I tried adding a specific `max-width` to the `additional-notes` class. Please let me know your thoughts!

**Review by maxxcrawford — APPROVED** [review]

**Pull request merged**